tests/Basketball/GameStats.java

```diff
@@ -0,0 +1,110 @@
+package Basketball;
+
+/*      - fouls
```

Comment:
Comments like this are fine for yourself, as you develop, but after implementing what is commented, it can safely be removed.